### PR TITLE
Rail Blog: Controller spec

### PR DESCRIPTION
**Pull request by Profsain**
### In this part of the rails project, I implement the following requirement.

- Created Controller spec for user action
- Created controller spec for posts action
- Pushed all changes to GitHub for pull requiest

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    it 'renders the index template' do
+      expect(response).to render_template('index')
+    end
+
+    it 'contains the correct text' do
+      expect(response.body).to include('List all users')
+    end
+  end
+
```
Comment (Davidosky007) [inline spec/requests/users_spec.rb]: * According to the project requirement 

> Make sure that for each action you check:

> If the response status was correct.
> If a correct template was rendered.
> If the response body includes correct placeholder text.
* Kindly ensure you write tests to check for all the above in all your actions as some are missing

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders the index template' do
+      expect(response).to render_template('index')
+    end
+  end
+
```
Comment (Davidosky007) [inline spec/requests/posts_spec.rb]: * According to the project requirement 

> Make sure that for each action you check:

> If the response status was correct.
> If a correct template was rendered.
> If the response body includes correct placeholder text.
* Kindly ensure you write tests to check for all the above in all your actions as some are missing

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Profsain 👋, 
almost-jumpin-gif

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Profsain 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**